scala-package/core/src/main/scala/ml/dmlc/mxnet/Initializer.scala

```diff
@@ -33,6 +33,10 @@ abstract class Initializer {
 
     if (name.startsWith("upsampling")) {
       initBilinear(name, arr)
+    } else if (name.startsWith("stn_loc") && name.endsWith("weight")) {
```

Comment:
@piiswrong implementation of converting initializer object to json string and back to object in Scala is a bit complex. I am still thinking a way to implement the same functionality of python's initializer in Scala. So for now the Scalapkg still use the legacy way to init.